ekip/nationalparks/data/pass-list-20160603.csv

```diff
@@ -544,8 +544,7 @@ Kisatchie NF - Winn District,318-628-4664,Winnfield,LA,http://www.fs.fed.us/r8/k
 Adams National Historical Park,617-770-1175,Quincy,MA,http://www.nps.gov/adam,YES,YES
 Assabet River NWR,978-562-3527,Sudbury,MA,http://www.fws.gov/refuge/assabet_river/,YES,YES
 Boston National Historical Park,617-242-5642,Boston,MA,http://www.nps.gov/bost,NO,YES
-Buffumville Lake,"508-248-5697
```

Comment:
Is this change related to the slugification? If not - maybe it should be a separate PR.